src/Hydra/Serializer/HydraPrefixTrait.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Hydra\Serializer;
+
+trait HydraPrefixTrait
+{
+    // Uncomment on API Platform 4
+    // const HYDRA_PREFIX = 'hydra:';
+    // const HYDRA_CONTEXT_HAS_PREFIX = 'hydra_prefix';
```

Comment:
Ok I see, but even in v4 it's a bit weird to "copy/paste" these constants in each class through a trait instead of having them in a single place.

Reply:
yes but for the subtree split it's almost mandatory I'll see if I can put  this only in once place

Reply:
Components that haven't a hard dependency on the Hydra component shouldn't depend on this constant (or should use `class_exist()` call).